packages/ses/NEWS.md

```diff
@@ -2,9 +2,23 @@ User-visible changes in `ses`
 
 # Next release
 
-- The `evalTaming:` option `safe-eval` now can only throw error `SES_DIRECT_EVAL`.
+- The [evalTaming](https://github.com/endojs/endo/blob/master/packages/ses/docs/lockdown.md#evaltaming-options)
+  option `safe-eval` now can only throw error `SES_DIRECT_EVAL`.
   This allows SES to initialize with `'unsafe-eval'` or `no-eval` on hosts with no
-  direct eval available (e.g. Hermes) for a successful lockdown.
+  direct eval available such as Hermes for a successful lockdown that tolerates it's
+  language [features](https://github.com/facebook/hermes/blob/main/doc/Features.md).
+
+  The module name `ses/hermes` can now be required to call `lockdown` and
+  `repairIntrinsics` only, `Compartment` is not yet available.
```

Comment:
was going to mention this will also be downloadable from
https://npmfs.com/download/ses/0.13.0/dist/ses-hermes.cjs

but https://npmfs.com/download/ses/0.12.0/dist/ses-hermes.cjs strangely doesn't work 🤔
https://npmfs.com/download/ses/0.12.0/dist/ses.cjs works as expected